poki-sdk/manifests/web/engine_template.html

```diff
@@ -1,28 +1,45 @@
 <!DOCTYPE html>
 <head>
-	<script src="//game-cdn.poki.com/scripts/v2/poki-sdk.js"></script>
+	<link rel="dns-prefetch" href="//game-cdn.poki.com" />
```

Comment:
I don't think this prefetch adds anything as the domain is already used 2 lines below. So browsers will see that at the same time and already start connecting to it.